applications/crossbar/doc/tasks.md

```diff
@@ -110,11 +119,15 @@ curl -v -X GET \
 }
 ```
 
+#### Add a new system task
+
 
 #### Add a new task
 
 > PUT /v2/accounts/{ACCOUNT_ID}/tasks
 
+Note: There are tasks that run against system resources, only for use by the superduper admin (like rate uploading), which can omit `/accounts/{ACCOUNT_ID}` from the URI. Leaving the account in the URI should have no impact.
+
```

Comment:
1. super admin is a crossbar construct, meaning the auth account id is flagged super duper. cb_tasks takes care of sending the auth account id to new/7 when there is no account id on the url path
1. The GET is added and this note should suffice. If people include the account id on the url, that's fine. This just hints they can leave it off if they want to save keystrokes.
1. Task module authors can add it. With rates, the eventual goal may be to allow resellers to upload their own ratedecks.

`extra_args()` is fine because new/7 is called with account IDs